ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.WebAPI/appsettings.Development.json

```diff
@@ -7,6 +7,7 @@
     }
   },
   "MongoDbSettings": {
-    "ConnectionString": "mongodb://localhost:27017/TimeManagementSystem" 
+    "ConnectionString": "mongodb://localhost:27017/TimeManagementSystem",
+    "CancellationTokenTimeOut":  0
```

Comment:
it's not MongoDbSetting, should be out of this section

Reply:
CancellationTokenTimeOut name really emphasizes cancellation token, but for app it's not important the way how we cancel, so it's better to name smth like TimeOutSeconds